scripts/governance/README.md

```diff
@@ -73,9 +73,21 @@ brownie run scripts/governance/create_proposal.py --network rsk-testnet
 brownie run scripts/governance/last_proposal_created.py --network rsk-testnet
 ```
 
+## Values.json
+
+- account : This is the account address used for checking the current voting power of that account, last proposal created by an account and also the account used for staking.
+- delegatee : The address of the delegatee (who will have the right to vote for the staked token).
```

Comment:
Is this referring to the address that will be able to vote on behalf of the account that I set in the `account` field? If so, I think the proper term would be "delgate" rather than "delegatee".